akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -12,6 +12,49 @@ To use Akka Cluster, you must add the following dependency in your project:
   version="$akka.version$"
 }
 
+## When and where to use Akka Cluster
+
+An architectural choice you have to make is if you are going to use a microservices architecture or
+a traditional distributed application, also known as a distributed monolith.
+
+Microservices has many attractive properties, such as the independent nature of microservices allows for
+multiple smaller and more focused teams that can deliver new functionality more frequently and can
+respond quicker to business opportunities. Reactive Microservices should be isolated, autonomous, and have
+a single responsibility as identified by Jonas Bonér in the book
+[Reactive Microservices Architecture: Design Principles for Distributed Systems](https://info.lightbend.com/COLL-20XX-Reactive-Microservices-Architecture-RES-LP.html).
+
```

Comment:
Maybe add: In a microservices system, you should consider communication within a service and between services.